src/locales/de/menu-ui-handler.ts

```diff
@@ -18,6 +18,10 @@ export const menuUiHandler: SimpleTranslationEntries = {
   "exportSlotSelect": "Wähle einen Slot zum Exportieren.",
   "importData": "Daten importieren",
   "exportData": "Daten exportieren",
+  "linkDiscord": "Link Discord",
+  "unlinkDiscord": "Unlink Discord",
+  "linkGoogle": "Link Google",
+  "unlinkGoogle": "Unlink Google",
```

Comment:
```suggestion
  "linkDiscord": "Discord verbinden",
  "unlinkDiscord": "Discord trennen",
  "linkGoogle": "Google verbinden",
  "unlinkGoogle": "Google trennen",
```

Honestly maybe those should be full sentences...
But i need to see see where they are used

Reply:
![image](https://github.com/pagefaultgames/pokerogue/assets/5632804/8a3a411e-5ceb-4e13-9290-0bd7dbe455a3)